packages/wonder-blocks-core/components/unique-id-provider.js

```diff
@@ -78,6 +77,11 @@ export default class UniqueIDProvider extends React.Component<Props> {
             return null;
         }
 
+        // Create an identifier factory if we don't already have one
+        if (!this._idFactory) {
+            this._idFactory = new UniqueIDFactory(scope);
```

Comment:
ASIDE: Should we consider a future update to the provider that allows for `scope` prop to change, and recreates the factory accordingly?

Reply:
That makes sense to me although I can't think of a situation where that would be useful.

Reply:
Fair; if we don't support it, we should probably call it out in the docs.